Slider/Assets/Scenes/Village.unity

```diff
@@ -438197,7 +438197,7 @@ MonoBehaviour:
     hasBeenAddedToDisplay: 0
   - hintData:
       hintName: tiles
-      hintText: Use the mouse to move sliders
+      hintText: Use the mouse or keyboard to move sliders
```

Comment:
i think we should leave this hint as mouse because i dont really want general people to be using keyboard for the artifact